frontend/src/lib/components/PayGateMini/payGateMiniLogic.tsx

```diff
@@ -36,11 +36,22 @@ export const payGateMiniLogic = kea<payGateMiniLogicType>([
         productWithFeature: [
             (s) => [s.billing],
             (billing) => {
-                // check addons first since their features are rolled up into the parent
+                // TODO(@zach): revisit this logic after subscribe to all products is released
+                // There are some features where we want to check the product first
+                const checkProductFirst = [AvailableFeature.ORGANIZATIONS_PROJECTS]
+
+                let foundProduct: BillingProductV2Type | BillingProductV2AddonType | undefined = undefined
+
+                if (checkProductFirst.includes(props.featureKey)) {
+                    foundProduct = billing?.products?.find((product) =>
+                        product.features?.some((f) => f.key === props.featureKey)
+                    )
+                }
+
+                // Check addons first if no included in the check list about since their features are rolled up into the parent
                 const allAddons = billing?.products?.map((product) => product.addons).flat() || []
-                let foundProduct: BillingProductV2Type | BillingProductV2AddonType | undefined = allAddons.find(
-                    (addon) => addon.features?.some((f) => f.key === props.featureKey)
-                )
+                foundProduct = allAddons.find((addon) => addon.features?.some((f) => f.key === props.featureKey))
```

Comment:
I think this will overwrite it with the found addon, which would land us back in the same spot. Needs `if (!foundProduct) {search addons}`

Reply:
Good catch, thank you!